dashboard/app/helpers/levels_helper.rb

```diff
@@ -26,9 +26,10 @@ def url_from_path(path)
   # @param [Hash] data Data to store in the channel.
   # @param [String] src Optional source channel to copy data from, instead of
   #   using the value from the `data` param.
-  def create_channel(data = {}, src = nil)
-
-    storage_app = StorageApps.new(storage_id('user'))
+  # @param [User] user Optional user to own the new channel.
+  def create_channel(data = {}, src = nil, user = nil)
```

Comment:
should we call this param owner instead of user? Otherwise we have two competing concepts of user.